unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -2766,7 +2766,10 @@ runScheme file args0 = do
   ensureSchemeExists
   gendir <- getSchemeGenLibDir
   statdir <- getSchemeStaticLibDir
-  let includes = gendir ++ ":" ++ statdir
+  let includes =
+        gendir ++ ":" ++
+        (statdir </> "common") ++ ":" ++
+        (statdir </> "chez")
```

Comment:
@dolio should this be `"racket"` rather than `"chez"`?

Reply:
Not until we're actually calling racket.